app/src/main/java/com/epfl/beatlink/model/navigation/AppIcons.kt

```diff
@@ -0,0 +1,354 @@
+package com.epfl.beatlink.model.navigation
+
+import androidx.compose.ui.geometry.Offset
+import androidx.compose.ui.graphics.Brush
+import androidx.compose.ui.graphics.SolidColor
+import androidx.compose.ui.graphics.vector.ImageVector
+import androidx.compose.ui.graphics.vector.ImageVector.Builder
+import androidx.compose.ui.graphics.vector.path
+import androidx.compose.ui.unit.dp
+import com.epfl.beatlink.ui.theme.PrimaryPurple
+import com.epfl.beatlink.ui.theme.PrimaryRed
+
+object AppIcons {
```

Comment:
The AppIcons object contains UI-specific resources like icons and gradients, so I think it would be better suited in the ui package to maintain clear separation of concerns.